frontend/src/components/taskSelection/action.js

```diff
@@ -90,6 +65,18 @@ export function TaskMapAction({ project, projectIsReady, tasks, activeTasks, act
     setActiveSection('history');
   };
 
+  const handleTaskHistories = (taskIds) => {
+    if (taskIds.length < 1) return;
+
+    taskIds.forEach((id) => {
+      if (Object.keys(multipleTasksInfo).indexOf(id.toString()) < 0) {
```

Comment:
`Object.keys(multipleTasksInfo).includes(id.toString())` is more expressive and straightforward.'